### DISCUSSION TITLE
Topic Reduction after Training Question

### DISCUSSION THREAD

**Discussion by jwegner99** [MaartenGr/BERTopic#2118] [General]
Hi Maarten,

Thank you so much for your help! I ran a model and have approximately 3500 topics. I used this line of code 
topic_model.reduce_topics(docs, nr_topics=50) to reduce the topics to 50. 

Now I am trying to (1) assign topics to my original dataset that I trained the model on of 500,000 docs and (2) transform the model onto an additional dataset of ~4,000,000 docs. Is there a way to find which of the 50 topics that the 500K dataset the model was trained on fit into? More specifically, is there a way to map the 3500 original topic list into the 50 reduced topics?

Thank you again!

**Comment by MaartenGr** [comment]
Thank you for the kind words!

After you have reduced the topics, you can access the updated topics with `topic_model.topics_`. 

To find out which of the 3500 original topics are reduced into the 50 reduced topics, you can access the `topic_model.topic_mapper_` for that I believe.
